dashboard/v1.1/src/router/Navigation.tsx

```diff
@@ -29,8 +29,8 @@ const Navigator: FC<NavigatorProps> = ({ updateLoader }) => {
         <Route path="/ping" component={PingModule} />
         <Route path="/floodping" component={FloodPing} />
         <Route path="/jitter" component={JitterModule} />
-        <Route path="/settings" component={Settings} />
         <Route path="/quick-input" component={Input} />
+        <Route path="/config" component={Config} />
```

Comment:
how about changing `/config` => `/configurations` ? Just from the user's perspective.